README.md

```diff
@@ -88,6 +88,19 @@ Store the Consul server configuration file in a ConfigMap:
 kubectl create configmap consul --from-file=configs/server.json
 ```
 
+### Create PVC and PV for Consul
+
+Create PersistentVolumeClaim and PersistentVolume which Consul uses to store data:
+
+```
+kubectl create -f persistenvolumes/data-consul-0-pv.yaml
```

Comment:
There is a typo here. It should be,
```
kubectl create -f persistentvolumes/data-consul-0-pv.yaml
kubectl create -f persistentvolumes/data-consul-0-pvc.yaml
kubectl create -f persistentvolumes/data-consul-1-pv.yaml
kubectl create -f persistentvolumes/data-consul-1-pvc.yaml
kubectl create -f persistentvolumes/data-consul-2-pv.yaml
kubectl create -f persistentvolumes/data-consul-2-pvc.yaml
```
Note  `persistenvolumes` => `persistentvolumes`

Reply:
Hey @tankchintan , typo has been fixed. Thanks for pointing that out!